<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delayLine_Dialog</class>
 <widget class="QDialog" name="delayLine_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>delayLine测试</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/img.qrc">
    <normaloff>:/new/prefix1/myappico.ico</normaloff>:/new/prefix1/myappico.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,20,0">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>delayLine功能测试</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>delayLine结果图：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="2" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>step2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_3" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>TDC_widget_3</zorder>
        <zorder>label_5</zorder>
        <zorder>label_TDC0_3</zorder>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>step3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_4" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>step1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_2" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>step4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_5" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>step0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>step5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_6" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_9" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>step7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_8" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,8,1">
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>step6</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCustomPlot" name="TDC_widget_7" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_TDC0_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,8,1,1,1,1">
     <item>
      <widget class="QPushButton" name="test_pushButton">
       <property name="text">
        <string>TEST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test_pushButton_2">
       <property name="text">
        <string>TEST</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>时钟相位(0x)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phase_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="getPhase_pushButton">
       <property name="text">
        <string>获取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delayLine_pushButton">
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/img.qrc"/>
 </resources>
 <connections/>
</ui>
